app/javascript/dashboard/assets/scss/_layout.scss

```
@@ -10,17 +10,32 @@ body {
 
 .app-wrapper {
   @include full-height;
+  flex-grow: 0;
+  min-height: 0;
   width: 100%;
 }
 
-.app-root {
+.banner + .app-wrapper {
+  .button--fixed-right-top {
+    top: 5.6 * $space-one;
+  }
```

```suggestion
.update-banner + .app-wrapper {
  .button--fixed-right-top {
    position: absolute;
    top: 5.6 * $space-one;
  }
```

Adding `position: absolute;` will solve this issue mentioned below in the screencast


https://user-images.githubusercontent.com/64252451/158370848-b830ebc9-2e30-43d6-b45b-7320338c9a50.mov

---

> Adding `position: absolute;` will solve this issue mentioned below in the screencast. And `.banner` is the class name of the banner component so it will affect the button when the other banner is active.


This only works if we are having the button inside the header component (eg - inbox settings).

---

Yes that's right, but I have tried in this case and it works.